src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionString.cs

```diff
@@ -20,176 +25,163 @@ internal sealed class SqlConnectionString : DbConnectionOptions
 
         internal static class DEFAULT
         {
-            private const string _emptyString = "";
             internal const ApplicationIntent ApplicationIntent = DbConnectionStringDefaults.ApplicationIntent;
-            internal const string Application_Name = TdsEnums.SQL_PROVIDER_NAME;
-            internal const bool Asynchronous = false;
-            internal const string AttachDBFilename = _emptyString;
-            internal const PoolBlockingPeriod PoolBlockingPeriod = DbConnectionStringDefaults.PoolBlockingPeriod;
-            internal const int Command_Timeout = ADP.DefaultCommandTimeout;
-            internal const int Connect_Timeout = ADP.DefaultConnectionTimeout;
-            internal const bool Connection_Reset = true;
-            internal const bool Context_Connection = false;
-            internal const string Current_Language = _emptyString;
-            internal const string Data_Source = _emptyString;
-            internal const bool Encrypt = true;
-            internal const bool Enlist = true;
-            internal const string FailoverPartner = _emptyString;
-            internal const string Initial_Catalog = _emptyString;
-            internal const bool Integrated_Security = false;
-            internal const int Load_Balance_Timeout = 0; // default of 0 means don't use
-            internal const bool MARS = false;
-            internal const int Max_Pool_Size = 100;
-            internal const int Min_Pool_Size = 0;
+            internal const string Application_Name = DbConnectionStringDefaults.ApplicationName;
+            internal const string AttachDBFilename = DbConnectionStringDefaults.AttachDBFilename;
```

Comment:
Not related to this PR, but the defaults are neither using `string.Empty` nor emptyString constant, they all have `""` as value. Can we fix those in a new PR, we probably left them as-is in #1265? @DavoudEshtehari